pkg/assets/assets.go

```diff
@@ -106,24 +119,22 @@ func assetFromTemplate(name string, tb []byte, data interface{}) (*Asset, error)
 
 type FileInfoPredicate func(path string, info os.FileInfo) (bool, error)
 
+type ManifestPredicate func(manifest []byte) (bool, error)
```

Comment:
Consider something like `FileContentsPredicate` to make it clearer for future users.